### PR TITLE
Infra cost reduction (Move from ECS to EC2)

### PR DESCRIPTION
**Pull request by LDMGN**

### CODE REVIEW

File: infrastructure/lib/api-ec2-stack.ts [hunk 1]
```diff
@@ -3,17 +3,21 @@ import {
...
     AmazonLinuxImage,
-    CfnEIP, CfnEIPAssociation, CfnKeyPair,
+    CfnEIP,
+    CfnEIPAssociation,
+    CfnKeyPair,
     Instance,
     InstanceClass,
     InstanceSize,
-    InstanceType, Peer, Port,
+    InstanceType, KeyPair,
```
Comment (MarcelWildenburg) [inline infrastructure/lib/api-ec2-stack.ts:12]: single import on one line ?

File: infrastructure/lib/api-ec2-stack.ts [hunk 2]
```diff
@@ -116,41 +160,46 @@ export class ApiEc2Stack extends Stack {
...
-        /* Logging. */
-        const apiEc2LoggingStack = new LoggingStack(this, 'ApiEc2LoggingStack', {
-            littil: props.littil,
-            logGroupName: 'BackendApiEc2Logs',
-        });
-
         /* Database access. */
         const databaseSecurityGroup = SecurityGroup.fromSecurityGroupId(this, 'DatabaseSecurityGroup', props.database.securityGroup.id);
-        databaseSecurityGroup.connections.allowFrom(ec2SecurityGroup, Port.tcp(parseInt(props.database.port)));
+        // databaseSecurityGroup.connections.allowFrom(ec2SecurityGroup, Port.tcp(parseInt(props.database.port)));
```
Comment (MarcelWildenburg) [inline infrastructure/lib/api-ec2-stack.ts:202]: do we need to keep this as comment ?

File: infrastructure/lib/database-stack.ts [hunk 3]
```diff
@@ -1,13 +1,14 @@
 import { CfnOutput, Stack, StackProps } from 'aws-cdk-lib';
-import { InstanceClass, InstanceSize, InstanceType, Vpc } from 'aws-cdk-lib/aws-ec2';
+import { InstanceClass, InstanceSize, InstanceType, SubnetType, Vpc } from 'aws-cdk-lib/aws-ec2';
+import { Secret } from 'aws-cdk-lib/aws-secretsmanager';
 import {
     Credentials,
     DatabaseInstance,
-    DatabaseInstanceEngine,
+    DatabaseInstanceEngine, DatabaseInstanceFromSnapshot,
```
Comment (MarcelWildenburg) [inline infrastructure/lib/database-stack.ts:7]: one import on single line ?
Check line 9 as well

**Review by MarcelWildenburg — APPROVED** [review]
some minor comments on formatting and unused import.

File: infrastructure/lib/api-ec2-stack.ts (hunk 2 above)
Comment (LDMGN) [reply infrastructure/lib/api-ec2-stack.ts:202]: Yes and no. It should be as limited as possible, but I don't have time to test it right now. So for now at least I'll add a TODO.

File: infrastructure/lib/database-stack.ts (hunk 3 above)
Comment (LDMGN) [reply infrastructure/lib/database-stack.ts:7]: Fixed by auto-formatting

File: infrastructure/lib/api-ec2-stack.ts (hunk 1 above)
Comment (LDMGN) [reply infrastructure/lib/api-ec2-stack.ts]: Fixed, and removed one unused import.

**Pull request merged**